teuthology/test/test_repo_utils.py

```diff
@@ -12,21 +13,24 @@
 
 
 class TestRepoUtils(object):
-    src_path = '/tmp/empty_src'
-    # online_repo_url = 'https://github.com/ceph/teuthology.git'
-    # online_repo_url = 'git://ceph.newdream.net/git/teuthology.git'
-    online_repo_url = 'https://github.com/ceph/empty.git'
-    offline_repo_url = 'file://' + src_path
-    repo_url = None
-    dest_path = '/tmp/empty_dest'
 
     @classmethod
     def setup_class(cls):
+        cls.temp_path = tempfile.mkdtemp(prefix='test_repo-')
+        cls.dest_path = cls.temp_path + '/empty_dest'
+        cls.src_path = cls.temp_path + '/empty_src'
+        cls.offline_repo_url = 'file://' + cls.src_path
```

Comment:
nit, no need to assign these variable as class variables. they could just be local variables.

Reply:
done, review